tests/integration/changeset/changeset.spec.ts

```diff
@@ -1,33 +1,42 @@
 import httpStatus, { StatusCodes } from 'http-status-codes';
-import { DependencyContainer } from 'tsyringe';
-import { Application } from 'express';
+import { container } from 'tsyringe';
 import faker from 'faker';
 import { Connection, QueryFailedError } from 'typeorm';
-import { postSync, getLatestSync } from '../sync/helpers/requestSender';
-import { postFile } from '../file/helpers/requestSender';
-import { postEntityBulk, patchEntities, patchEntity, postEntity } from '../entity/helpers/requestSender';
-import { registerTestValues } from '../testContainerConfig';
 import { createStringifiedFakeFile } from '../file/helpers/generators';
 import { createStringifiedFakeSync } from '../sync/helpers/generators';
 import { createStringifiedFakeEntity } from '../entity/helpers/generators';
 import { EntityStatus, Status } from '../../../src/common/enums';
 import { Sync } from '../../../src/sync/models/sync';
-import * as requestSender from './helpers/requestSender';
+import { getApp } from '../../../src/app';
+import { EntityRequestSender } from '../entity/helpers/requestSender';
+import { FileRequestSender } from '../file/helpers/requestSender';
+import { SyncRequestSender } from '../sync/helpers/requestSender';
+import { Services } from '../../../src/common/constants';
+import { changesetRepositorySymbol } from '../../../src/changeset/DAL/changsetRepository';
+import { TransactionFailureError } from '../../../src/changeset/models/errors';
+import { getBaseRegisterOptions } from '../helpers';
+import { IApplication } from '../../../src/common/interfaces';
+import { ChangesetRequestSender } from './helpers/requestSender';
 import { createStringifiedFakeChangeset } from './helpers/generators';
 
 jest.setTimeout(3000000);
```

Comment:
can be removed from `entity.spec.ts`.
in `changeset.spec.ts` the Flow integration tests are taking longer than the default jest async callback timeout length which is 5000 ms.
the given value to `jest.setTimeout` of 3000000 (5 minutes) can be lowered to 10000 (10 seconds).
also, `runInBand` option can be removed. this will make the tests run faster.

Reply:
add to each flow test the appropriate timeout, if we do it at a module level, we lower the bar for other tests which we would like them not to timeout as it might indicate an error.

Reply:
done.